violetproduct-swing/src/main/java/com/horstmann/violet/application/gui/SplashFrame.java

```diff
@@ -0,0 +1,329 @@
+import com.horstmann.violet.framework.injection.bean.ManiocFramework.InjectedBean;
+import com.horstmann.violet.framework.injection.resources.ResourceBundleInjector;
+import com.horstmann.violet.framework.injection.resources.annotation.ResourceBundleBean;
+import com.horstmann.violet.framework.plugin.IDiagramPlugin;
+import com.horstmann.violet.framework.plugin.PluginRegistry;
+import com.horstmann.violet.framework.userpreferences.UserPreferencesService;
+import com.horstmann.violet.product.diagram.abstracts.IGraph;
+import com.horstmann.violet.workspace.IWorkspace;
+import com.horstmann.violet.workspace.Workspace;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * This desktop frame contains panes that show menu to create/open projects.
+ *
+ * @author Maciej Kupniewski
+ */
+public class SplashFrame {
+
+    private JButton btnNewProject;
+    private JButton btnOpenProject;
+    private JButton btnOpenRecent;
+    private JButton btnRecentProject;
+
+    private JButton btnClassDiagram;
+    private JButton btnComunicationDiagram;
+    private JButton btnSequenceDiagram;
+    private JButton btnStateDiagram;
+    private JButton btnUseCaseDiagram;
+    private JButton btnObjectDiagram;
+    private JButton btnActivityDiagram;
+
+    private JPanel welcomePanel = new JPanel();
+    private JPanel openRecentPanel = new JPanel();
+    private JPanel mainPanel = new JPanel();
+    private JPanel viewsPanel = new JPanel();
+
+    private CardLayout cardLayout = new CardLayout();
+    private JFrame frame = new JFrame();
+    private List<IDiagramPlugin> diagramPlugins;
+
+    /**
+     * Constructs frame with 3 buttons to create/open or open recent project.
+     *
+     * @param mainFrame it's for add workspace to main frame.
+     */
+    public SplashFrame(final MainFrame mainFrame) {
+        mainPanel.setLayout(cardLayout);
+        frame.setTitle("Okno Powitalne");
+        this.mainFrame = mainFrame;
+        ResourceBundleInjector.getInjector().inject(this);
+        BeanInjector.getInjector().inject(this);
+        diagramPlugins = pluginRegistry.getDiagramPlugins();
+
+        frame.setSize(300, 300);
+        frame.setResizable(false);
+        btnNewProject = new JButton(btnNameNewProject);
+        btnOpenProject = new JButton(btnNameOpenProject);
+        btnOpenRecent = new JButton(btnNameOpenRecent);
+
+        btnActivityDiagram = new JButton(btnNameActivityDiagram);
+        btnClassDiagram = new JButton(btnNameClassDiagram);
+        btnComunicationDiagram = new JButton(btnNameComunicationDiagram);
+        btnSequenceDiagram = new JButton(btnNameSequenceDiagram);
+        btnStateDiagram = new JButton(btnNameStateDiagram);
+        btnUseCaseDiagram = new JButton(btnNameUseCaseDiagram);
+        btnObjectDiagram = new JButton(btnNameObjectDiagram);
+
```

Comment:
this not working with feature 'settings language in project'.. always shows in polish, try fix it